test/runtests.py

```diff
@@ -270,30 +275,31 @@ def _wait_for_binding_delete(self, queue_name, exchange_name, routing_key):
                 Binding.routing_key == routing_key).first():
                 break
 
-    def test_abnormal_queue_name(self):
-        self.consumer_class = AbnormalQueueConsumer
+    def test_reserved_queue_name(self):
```

Comment:
Shouldn't you be adding a new test instead of replacing this test?  I think we still want to keep ``test_abnormal_queue_name`` don't we?